module/Frontpage/view/frontpage/frontpage/home.phtml

```diff
@@ -99,7 +99,7 @@
                         <div class="list-group">
                             <?php foreach ($activities as $activity): ?>
                                 <a class="list-group-item" href="<?= $this->url('activity/view', ['id' => $activity->getId()]) ?>">
-                                    <h4 class="list-group-item-heading"><?= $activity->getName() ?></h4>
+                                    <h4 class="list-group-item-heading"><?= $activity->getNameEn() ?></h4>
```

Comment:
Why is it forcing the English translation?

Reply:
Because previously it was forcing the Dutch translation which isn't always available. This will be fixed properly once #801 is fixed.

Reply:
In that case, I approve this hack!